packages/models-library/src/models_library/invitations.py

```diff
@@ -19,14 +19,19 @@ class InvitationInputs(BaseModel):
         description="Invitee's email. Note that the registration can ONLY be used with this email",
     )
     trial_account_days: PositiveInt | None = Field(
-        None,
+        default=None,
         description="If set, this invitation will activate a trial account."
         "Sets the number of days from creation until the account expires",
     )
     extra_credits_in_usd: PositiveInt | None = Field(
-        None,
+        default=None,
         description="If set, the account's primary wallet will add extra credits corresponding to this ammount in USD",
     )
+    product: str | None = Field(
```

Comment:
Would it make sense to make this an `enum` instead of a `str`, or maybe a `Literal`?

Reply:
@bisgaard-itis this input is for the `product_name` which is a unique identifier. We (z43) manage a finite set of products with given `product_name`s but in principle the products are not constraint to those.

Reply:
thanks for the clarification